### PR TITLE
How to update to 6.0.10 - SharedFolder

### PR DESCRIPTION
**Pull request by trasherdk**
The patch for `endpoints/lib/vboxServiceWrappers.php` was not supposed to be in the PR, but I have no idea how to remove it. Sorry :)

### CODE REVIEW

File: update-6.0.10-linux.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+Let's say I have `phpvirtualbox` installed in `~/phpvirtualbox`.
+```bash
+cd ~
+wget https://download.virtualbox.org/virtualbox/6.0.10/VirtualBoxSDK-6.0.10-132072.zip
+unzip VirtualBoxSDK-6.0.10-132072.zip
+mv sdk sdk-6.0.10
+cp sdk-6.0.10/bindings/webservice/vboxweb.wsdl phpvirtualbox/endpoints/lib/vboxweb-6.0.wsdl
+cp sdk-6.0.10/bindings/webservice/php/lib/vboxServiceWrappers.php phpvirtualbox/endpoints/lib/
+```
```
Comment (trasherdk) [inline update-6.0.10-linux.md]: I forgot:
```
cp sdk-6.0.10/bindings/webservice/vboxwebService.wsdl phpvirtualbox/endpoints/lib/vboxwebService-6.0.wsdl
```

**Comment by chotaire** [comment]
This PR should break functionality (snapshotting), changes things very specific to your installation (e.g. path to VBoxGuestAdditions.iso) and touches files that have nothing to do with the actual update. You added new documentation for a very specific version of VirtualBox SDK (6.0.10) which is very confusing because from what I see you have already committed the changes explained in your update howto, so there should be no need for a howto? There is also zero reference in your PR that the actual code is - if I am not mistaken - by @maxcalandrelli. One of your commits is mentioning that you are updating generated files from SDK 6.0.10, may I ask what/how files were generated? If there is a generator that adds phpVirtualBox specific changes, please shed some light.

**Comment by trasherdk** [comment]
How would this break `snapshotting`?

All changes are made on a [shared-folder branch](https://github.com/trasherdk/phpvirtualbox/tree/shared-folder)

When installing [VirtualBox-6.0.10](https://download.virtualbox.org/virtualbox/6.0.10/VirtualBox-6.0.10-132072-Linux_amd64.run), and earlier versions,, the destination is `/opt/VirtualBox` and the path to `VBoxGuestAdditions.iso` will be `/opt/VirtualBox/additions/VBoxGuestAdditions.iso`.

The previous path, to the `VBoxGuestAdditions.iso`, was `/usr/share/virtualbox`, which does not exist.

Generated files, `vboxServiceWrappers.php, vboxweb.wsdl and vboxwebService.wsdl`  are from [VirtualBoxSDK-6.0.10](https://download.virtualbox.org/virtualbox/6.0.10/VirtualBoxSDK-6.0.10-132072.zip) and not generated by me. I just added `6.0` to the filename, and changed the `<import location="vboxweb-6.0.wsdl" namespace="http://www.virtualbox.org/"/>` to point to the new `.wsdl` file.

The `howto` is documenting the steps I took to do the `version upgrade`.

**Comment by trasherdk** [comment]
Anyway. This PR is so far behind the current state of `shared-folder` branch, that it would probably not work.
Closing....

**Pull request closed without merging**